Jenkinsfile_CNP

```diff
@@ -175,6 +175,46 @@ withPipeline(type, product, component) {
         enablePactAs([AppPipelineDsl.PactRoles.CONSUMER])
                         //Disabling due to IDAM OIDC issue. To be investigated under SSCS-10377
                       //AppPipelineDsl.PactRoles.CONSUMER_DEPLOY_CHECK])
+
+        env.ENVIRONMENT = "pr"
+        for (label in githubApi.getLabelsbyPattern(env.BRANCH_NAME, "pr-values:ccd") ) {
```

Comment:
will this be removed before merge? or will we need to put a label on our PRs each time

Reply:
that's how it currently works with the preview environment. It won't deploy the ccd stack unless you add that. We are looking to change it so that it becomes the default